arq/worker.py

```diff
@@ -192,6 +196,7 @@ def __init__(
         on_job_end: Optional['StartupShutdown'] = None,
         after_job_end: Optional['StartupShutdown'] = None,
         handle_signals: bool = True,
+        wait_for_job_completion_on_signal_seconds: int = 0,
```

Comment:
Feel free to suggest a better name here too.

Reply:
probably too long, but not enough for me to bother changing it now :smile:.

Reply:
`job_completion_wait`??? Should be descriptive enough with a good docstring?